Dvd updates (#6805)
google/WebFundamentals: src/content/en/shows/designer-vs-developer/podcast/DVDS3E02-designing-for-standards.md

project_path: /web/_project.yaml
book_path: /web/shows/_book.yaml

> description: Bruce Lawson talks about designing for standards, and not just one browser. We also cover Styleable, a CSS preprocessor that they are working on at Wix.

Reviewer: ```suggestion
description: Bruce Lawson talks about designing for standards, and not just one browser. We also cover Stylable, a CSS preprocessor developed at Wix.
```

https://stylable.io
